server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/main/java/org/apache/james/webadmin/integration/vault/DeletedMessageVaultIntegrationTest.java

```diff
@@ -481,7 +481,7 @@ void vaultEndpointShouldRestoreMessageForSharer() {
 
     @Tag(BasicFeature.TAG)
     @Test
-    void vaultExportShouldExportZipContainsVaultMessagesToShareeWhenJmapDeleteMessage() throws Exception {
+    public void vaultExportShouldExportZipContainsVaultMessagesToShareeWhenJmapDeleteMessage() throws Exception {
```

Comment:
Why putting public this one if it's not to disable it after in `RabbitMQDeletedMessageVaultIntegrationTest`?

Reply:
To override it.

Reply:
Yes but it's not being overriden :)